ruby/forwardfinancing.rb

```diff
@@ -75,7 +78,7 @@
   },
   body: json
 )
-
+puts response
```

Comment:
Add something to distinguish these `puts` for what they are. For example `puts "response: #{response}"`.

Reply:
done